File: Programmers/LV1/내적/내적.py
# 프로그래머스 내적 LV1
def solution(a,b):
    answer = 1234567890
    res=0

    for i in range(len(a)):
        res += a[i] *b[i]


    answer = res
    return answer



a = [1,2,3,4]

b = [-3,-1,0,2]

print(solution(a,b))

File: Programmers/LV1/JAVA/내적.java
package Programmers.level1;

public class 내적 {

	public static void main(String[] args) {
		int[] a = { 1, 2, 3, 4 };
		int[] b = { -3, -1, 0, 2 };

		System.out.println(solution(a, b));
	}

	public static int solution(int[] a, int[] b) {
		int answer = 0;
		int n = a.length;
		for (int i = 0; i < n; i++) {
			answer += (a[i] * b[i]);
		}

		return answer;
	}

}
